### ISSUE TITLE
HICO-DET finetuned Detr

### ISSUE THREAD

**Issue by jihwanp** [YueLiao/CDN#2]
HI thank u for sharing such a nice work!

I currently use HOI models for custom datasets and I wonder if u have plans to share pretrained DETR with fine-tuned on HICO-DET. 

Thanks

**Comment by YueLiao** [comment]
Thx for your interest in our work~

For a fair comparison, we have fine-tuned the DETR with a 3-layer decoder on HICO-DET. This is this [checkpoint](https://drive.google.com/file/d/1uho38sp1rTQUoiQC-K_t1B6PJ0TJaeXs/view?usp=sharing).

**Comment by jihwanp** [comment]
Thanks for sharing!

I have one more question about evaluation. 
What's the main difference between `map_thesis` and official metric `sc1`? 
It seems `map_thesis` is a little higher than `sc1,` and I'm curious why not use `sc1` to evaluate during training.

**Comment by YueLiao** [comment]
The official evaluation pipeline of HICO-Det and v-coco has a very complex process, e.g., a complex prediction format transformation or time-consuming. Thus, previous works (PPDM, QPIC) provide a similar and easy way to evaluate the results to obtain a real-time reward. However, we still need to use the official script to evaluate the final result for a fair comparison.

**Comment by SISTMrL** [comment]
@YueLiao Hello, YueLiao, Could you provide the official evaluation code on HICODET, I want to know the input format on official matlab code, thanks!

**Comment by YueLiao** [comment]
> @YueLiao Hello, YueLiao, Could you provide the official evaluation code on HICODET, I want to know the input format on official matlab code, thanks!

We follow PPDM to evaluate the experimental results through the official Matlab code. And we will upload the scripts transforming the CDN results for the Matlab evaluation format later.

**Comment by YueLiao** [comment]
> @YueLiao Hello, YueLiao, Could you provide the official evaluation code on HICODET, I want to know the input format on official matlab code, thanks!

https://gist.github.com/YueLiao/93774b9698bd06c42aa85de35f7ff190 with this script and the eval in PPDM. @SISTMrL

**Comment by YueLiao** [comment]
If still have any questions, feel free to reopen.
